M/MuJoCo/build_tarballs.jl

```diff
@@ -0,0 +1,52 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "MuJoCo"
+version = v"2.3.7"
+
+# Collection of sources required to build libX11
+sources = [
+    GitSource("https://github.com/deepmind/mujoco", "c9246e1f5006379d599e0bcddf159a8616d31441")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/mujoco
+sed -i '/Werror/d' cmake/MujocoOptions.cmake
+CPPFLAGS="-I${prefix}/include"
+CXXFLAGS="-I${prefix}/include"
+apk update
+apk upgrade
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release .
+cmake --build .
+cmake --install .
+"""
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [p for p in supported_platforms() if !Sys.isapple(p) || p.tags["arch"] != "aarch64"]
```

Comment:
The upstream repository ships macOS binaries as well as aarch64 binaries, so why are they excluded here? Compilation trouble?

Reply:
Yeah, we ran into compilation errors with aarch on Apple, and we plan to come back to this once we have some updated bindings.